static/app/views/performance/trends/utils.tsx

```diff
@@ -236,13 +236,12 @@ export function modifyTrendView(
   location: Location,
   trendsType: TrendChangeType,
   projects: Project[],
-  organization: OrganizationSummary,
-  isProjectOnly?: boolean
+  organization: OrganizationSummary
 ) {
   const trendFunction = getCurrentTrendFunction(location);
   const trendParameter = getCurrentTrendParameter(location, projects, trendView.project);
 
-  const transactionField = isProjectOnly ? [] : ['transaction'];
+  const transactionField = ['transaction'];
   const fields = [...transactionField, 'project'].map(field => ({
```

Comment:
small nit: we can just inline `'transaction'` into `fields` now, right?

Reply:
```suggestion
  const fields = ['transaction', 'project'].map(field => ({
```
I probably should've just left a change suggestion :)

Reply:
done!